File: python/071924_B4_24087.py
# https://www.acmicpc.net/problem/24087
sab = []
for i in range(3):
    sab.append(int(input()))
ret = 250
if sab[0] > sab[1] :
    ret += ((sab[0] - sab[1] - 1) // sab[2] + 1) * 100
print(ret)

File: cpp/071924_B4_24087.cpp
/**
 * https://www.acmicpc.net/problem/24087
 */
#include <iostream>
using namespace std;

int main() {
	int s, a, b;
	int ret = 250;
	
	cin >> s >> a >> b;
	if( s > a ) {
		ret += (((s - a) -1) / b + 1) * 100;
	}
	cout << ret << endl;
}
